### ISSUE TITLE
Implement a repeat function

### ISSUE THREAD

**Issue by mpatkisson** [redbarn-io/redbarn#1]
Create some sort of function which allows the end user to easily bind a set of repeated items.  The end user should be able to type something like this:

````javascript
  var items = [
    { foo: 'foo 1', bar: 'bar 1', baz: 'baz 1' },
    { foo: 'foo 2', bar: 'bar 2', baz: 'baz 2' },
    { foo: 'foo 3', bar: 'bar 3', baz: 'baz 3' },
  ];

  $('ul > li').repeat(items, function (item, li) {
    li.find('.foo').text(item.foo);
    li.find('.bar').text(item.bar);
    li.find('.baz').text(item.baz);
  });
````

**Comment by mpatkisson** [comment]
I've created an entirely seperate git repo to assist with extending cheerio and lodash.  In addition, it allows a clean way to add things to Nashorn all through a single bundled script.  The project is called redbarn-nashorn-bundler.  I've currently written and tested a repeat extension to cheerio which functions as described above.

**Comment by mpatkisson** [comment]
This is partially resolved with 378d03a, though I still need to provide a test for it.
